contrib/ros/README.md

```diff
@@ -0,0 +1,138 @@
+# rllab.contrib.ros Instructions
+## Introduction
+Most of the current environments for reinfocement learning training(e.g. gym's environments) are not particularly 
+designed for robot learning. Consequently, most of their environments are not for continuous action space task. And 
+it is also hard for robot learning researchers to use the same environment interface when they want to test their 
+algorithms on real robots. 
+
+rllab.contrib.ros's environments take a different approach. It's targeting at provide the same interfaces for both of the 
```

Comment:
It's targeted towards providing the same interface for training both simulated and real-life robots.